views/status.jade

```diff
@@ -0,0 +1,115 @@
+extends ./layout.jade
+
+block scripts
+    <!-- CSS includes -->
+    <!-- Dynatable styling -->
+    link(rel='stylesheet' href='/css/jquery.dynatable.css')
+    
+    <!-- CSS for some things in dynatable. -->
+    link(rel="stylesheet" href="/css/pages.css")
+    
+    <!-- Javascript includes -->
+    <!-- API that allows for JSON objects for creating paginated tables. -->
+    script(src="/js/jquery.dynatable.js")
+    
+    <!-- Character counter for song title and step artist search. -->
+    script(src='/js/character_count.js')
+    
+    <!-- Accordion collapse behavior script. Transitions between Song Title Search and Step Artist Search. -->
+    script(src="/js/accordion_focus.js")
+    
+    <!-- AutoComplete Table for Song Title Search -->
+    script(src="/js/autocomplete_songname.js")
+    
+    <!-- AutoComplete Table for Step Artist Search -->
+    script(src="/js/autocomplete_stepartist.js")
+    
+    <!-- Validation of input text fields for Song Title and Step Artist -->
+    script(src="/js/search_validate.js")
+    
+    <!-- Resizable Table relative to Browser Window -->
+    link(rel="stylesheet" href="/css/resizable_Table.css")
+    
+    script.
+        $(document).ready( function () { initCharCounterSongTitle(); initCharCounterStepArtist(); initAccordionFocusSettings(); autocompleteSongTitle(); autocompleteStepArtist(); });
+
+block content
```

Comment:
Is it possible to just make an array of possible statuses (Accepted, CQ, etc) and just iterate through those and execute the same code block? Would be shorter that way